anime-list-master.xml

```diff
@@ -20690,7 +20690,7 @@
     <name>Code Geass: Boukoku no Akito</name>
     <mapping-list>
       <mapping anidbseason="0" tvdbseason="0">;1-0;2-0;3-0;4-0;</mapping>
-      <mapping anidbseason="1" tvdbseason="0">;1-34;2-37;3-41;4-43;5-45;</mapping>
+      <mapping anidbseason="1" tvdbseason="0" start="1" end="5" offset="24" />
```

Comment:
Were the specials changed again?

The TVDB episodes are now 25, 28, 32, 33, 34, 36.

Reply:
So it seems... More Picture Dramas added in-between *sigh*